test/Intrinsics/fshlr.ll

```diff
@@ -0,0 +1,75 @@
+; Tests for the Funnel Shift intrinsics fshl/fshr(a, b, c) which
```

Comment:
You also need `; REQUIRES: geq-llvm-7.0` to only enable this test for later LLVM versions. Sorry for not spotting it earlier.

Reply:
Of course - I should also have spotted this.
Thanks for the directly actionable comments!